src/js/hooks/cycleCount/countStep/useCycleCountImport.js

```diff
@@ -0,0 +1,41 @@
+import { useCallback, useState } from 'react';
+
+import { useDispatch, useStore } from 'react-redux';
+
+import useSpinner from 'hooks/useSpinner';
+import { importCycleCounts } from 'utils/cycleCountUtils';
+
+const useCycleCountImport = (currentLocationId, locale) => {
+  const [importErrors, setImportErrors] = useState([]);
+  const dispatch = useDispatch();
+  const store = useStore();
```

Comment:
I can see you starting to use `useStore` - could you explain the reason for that, so not to use the `useSelector`?
I found one article regarding that and one comment surprised me: 
![Uploading Screenshot From 2025-11-24 14-40-27.png…]()

Reply:
The idea of that was to access the data only in function; you can see that I am working on the store only inside 1-2 functions. I wanted to avoid subscribing to any data that can cause re-renders, and it's used only, for example, while importing data. In that case, useSelectors will result in re-rendering, because shallow comparison will return false.